src/modules/users/users.service.ts

```diff
@@ -673,10 +673,8 @@ export class UserService {
         data: response,
       });
     } catch (error) {
-      return new ErrorResponse({
```

Comment:
@Xitija Do you know if removing statusCode from api response is breaking UI anywhere?